src/tensors/tensor.jl

```diff
@@ -559,18 +566,22 @@ function Base.show(io::IO, t::TensorMap)
     if sectortype(t) == Trivial
         Base.print_array(io, t[])
         println(io)
-    elseif FusionStyle(sectortype(t)) isa UniqueFusion
-        for (f₁, f₂) in fusiontrees(t)
-            println(io, "* Data for sector ", f₁.uncoupled, " ← ", f₂.uncoupled, ":")
-            Base.print_array(io, t[f₁, f₂])
-            println(io)
-        end
     else
-        for (f₁, f₂) in fusiontrees(t)
-            println(io, "* Data for fusiontree ", f₁, " ← ", f₂, ":")
-            Base.print_array(io, t[f₁, f₂])
-            println(io)
+        for (c, b) in blocks(t)
+            println(io, "* Data for sector $c:")
+            Base.print_array(io, b)
         end
+        #     for (f₁, f₂) in fusiontrees(t)
+        #         println(io, "* Data for sector ", f₁.uncoupled, " ← ", f₂.uncoupled, ":")
+        #         Base.print_array(io, t[f₁, f₂])
+        #         println(io)
+        #     end
+        # else
+        #     for (f₁, f₂) in fusiontrees(t)
+        #         println(io, "* Data for fusiontree ", f₁, " ← ", f₂, ":")
+        #         Base.print_array(io, t[f₁, f₂])
+        #         println(io)
+        #     end
```

Comment:
Is this a temporary change or would you like to change printing for `TensorMap`s?

Reply:
I personally prefer printing the big blocks, simply because it is a lot more compact. Inspecting the contents of two tensors is quite annoying with the fusiontrees in the way. Here I just temporarily changed it because I was debugging and forgot to change it back, I would definitely do that in a separate PR anyways